base/util.jl

```diff
@@ -509,6 +509,32 @@ function _kwdef!(blk, params_args, call_args)
     blk
 end
 
+"""
+    @invokelatest f(args...; kwargs...)
+
+Provides a convenient way to call [`Base.invokelatest`](@ref).
+`@invokelatest f(args...; kwargs...)` will simply be expanded into
+`Base.invokelatest(f, args...; kwargs...)`.
+"""
+macro invokelatest(ex)
+    @assert is_expr(ex, :call) "call expression f(args...; kwargs...) should be given"
```

Comment:
This shouldn't be an assert; it should throw an ArgumentError.

Reply:
done